prediction_market_agent/agents/microchain_agent/app.py

```diff
@@ -124,7 +125,9 @@ def long_term_memory_is_initialized() -> bool:
 def maybe_initialize_long_term_memory() -> None:
     # Initialize the db storage
     if not long_term_memory_is_initialized():
-        st.session_state.long_term_memory = LongTermMemoryTableHandler(AGENT_IDENTIFIER)
+        st.session_state.long_term_memory = (
+            LongTermMemoryTableHandler.from_agent_identifier(AGENT_IDENTIFIER)
```

Comment:
We have a little identity issue with how to identify agents.

`AgentIdentifier` class exists, but in `LongTermMemoryTableHandler` it was passed into `task_description` argument, in `PromptTableHandler` it was passed to `session_identifier` argument., Somewhere it's used as string, somewhere as enum. Trying to consolidate it at least a little.